spec/abci/abci++_methods.md

```diff
@@ -797,7 +793,7 @@ Most of the data structures used in ABCI are shared [common data structures](../
     `Metadata`). Chunks may be retrieved from all nodes that have the same snapshot.
     * When sent across the network, a snapshot message can be at most 4 MB.
 
-## Data types introduced or modified in ABCI++
+## Data types introduced or modified in ABCI 2.0
```

Comment:
The previous section has title: Data Types existing in ABCI

Is this legacy ABCI (<1) or ABCI 1.0?

Reply:
could be both (so maybe just say "Data Types existing before ABCI 2.0")